spec/features/translations_spec.rb

```diff
@@ -71,6 +71,18 @@
       expect(page).not_to have_link "Español"
     end
 
+    scenario 'Change value of a translated field to blank' do
+      milestone.update_attributes!(status: create(:budget_investment_status))
+      visit @edit_milestone_url
+
+      fill_in 'budget_investment_milestone_description_en', with: ''
+
+      click_button "Update milestone"
+      expect(page).to have_content "Milestone updated successfully"
+
+      expect(milestone.reload.description).to be_blank
```

Comment:
I think usually we don't check database records directly in feature specs. How about something like:
```ruby
visit @edit_milestone_url
expect(page).to have_content "Description in English"

fill_in 'budget_investment_milestone_description_en', with: ''

click_button "Update milestone"
expect(page).to have_content "Milestone updated successfully"
expect(page).not_to have_content "Description in English"

```

Reply:
Okay, I'll avoid that. Changing the test.

Reply:
Fixed 5ac8467